packages/cursorless-engine/src/processTargets/modifiers/VisibleStage.ts

```diff
@@ -0,0 +1,19 @@
+import { VisibleModifier } from "@cursorless/common";
+import { Target } from "../../typings/target.types";
+import { ModifierStage } from "../PipelineStages.types";
+import { PlainTarget } from "../targets";
+
+export class VisibleStage implements ModifierStage {
+  constructor(private modifier: VisibleModifier) {}
```

Comment:
not worth changing for this PR, esp since we do this all over the place, but don't feel compelled to pass in modifier if it's not actually used. Gives a nice signal we don't use it if we don't even pass it in. I think one of the benefits of our DI setup is that we don't enforce uniformity like other DI mechanisms

Reply:
Yeah I guess that's fair; hadn't thought bout that